mail_thread_create_nolog/tests/test_create_nolog.py

```diff
@@ -1,10 +1,10 @@
 # Copyright 2023 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import TransactionCase
 
 
-class TestMailThreadCreateNoLog(SavepointCase):
+class TestMailThreadCreateNoLog(TransactionCase):
```

Comment:
As the module now overrides the std `mail` controller to inject the fake message, should we now test a call on this one instead? Leveraging `odoo.tests.comm.HttpCase`.

What do you think?